.rubocop.yml

```diff
@@ -5,7 +5,7 @@ require:
 - rubocop-rspec
 
 AllCops:
-  TargetRubyVersion: 2.7
+  TargetRubyVersion: 3.4
```

Comment:
That should match the lowest supported ruby version in the gemspec. Otherwise we might break compatibility with rubies we still support